src/sentry/static/sentry/app/utils/discover/fieldRenderers.tsx

```diff
@@ -258,6 +260,53 @@ const SPECIAL_FIELDS: SpecialFields = {
   },
 };
 
+type SpecialFunctions = {
+  user_misery: SpecialField;
+};
+
+/**
+ * "Special functions" are functions whose values either do not map 1:1 to a single column,
+ * or they require custom UI formatting that can't be handled by the datatype formatters.
+ */
+const SPECIAL_FUNCTIONS: SpecialFunctions = {
+  user_misery: {
+    sortField: null,
+    renderFunc: data => {
+      const uniqueUsers = data.count_unique_user;
+      let userMiseryField;
+      for (const field in data) {
+        if (field.startsWith('user_misery')) {
+          userMiseryField = field;
+        }
+      }
+      if (!userMiseryField) {
+        return <NumberContainer>{emptyValue}</NumberContainer>;
+      }
+
+      const userMisery = data[userMiseryField];
+
+      if (!uniqueUsers && uniqueUsers !== 0) {
+        return (
+          <NumberContainer>
+            {typeof userMisery === 'number' ? formatFloat(userMisery, 4) : emptyValue}
+          </NumberContainer>
+        );
+      }
+
+      const palette = new Array(10).fill('#1a1772');
```

Comment:
I don't know the right colour for this, any tips would be handy. I did VERY little formatting of this, so it might need some touching up.

Reply:
@doralchan

Reply:
I would start off with one of the theme gray/purples and go from there.